backend/setup.py

```diff
@@ -81,6 +81,9 @@
     "webpreview @ git+https://github.com/algoo/webpreview@v1.6.0+algoo",
     # importlib
     "importlib_metadata==4.6.0",
+    # note pdf preview
+    "pypandoc",
+    "weasyprint<53",
```

Comment:
weasyprint 53 and later doesn't work on ubuntu 18.04 and debian buster because it require Pango >= 1.44.0.

Reply:
for information, the erreur obtained:
```
AttributeError: function/symbol 'pango_context_set_round_glyph_positions' not found in library 'libpango-1.0.so.0': /usr/lib/x86_64-linux-gnu/libpango-1.0.so.0: undefined symbol: pango_context_set_round_glyph_positions
Error producing PDF
```